apps/climatemappedafrica/src/components/Hero/Hero.js

```diff
@@ -0,0 +1,186 @@
+import { RichTypography } from "@commons-ui/core";
+import { Box, Grid, useMediaQuery } from "@mui/material";
+import { useTheme } from "@mui/material/styles";
+import dynamic from "next/dynamic";
+import PropTypes from "prop-types";
+import React, { useState } from "react";
+
+import heroBg from "@/climatemappedafrica/assets/images/bg-map-white.jpg";
+import DropdownSearch from "@/climatemappedafrica/components/DropdownSearch";
+import Image from "@/climatemappedafrica/components/Image";
+import RichHeader from "@/climatemappedafrica/components/RichHeader";
+import Section from "@/climatemappedafrica/components/Section";
+
+const Map = dynamic(() => import("./Map"), { ssr: false });
+
+function Hero({
+  comment,
+  title,
+  subtitle,
+  searchLabel,
+  featuredLocations,
+  searchPlaceholder,
+  properties,
+  location: { center },
+  level,
+  ...props
+}) {
+  const theme = useTheme();
+  const { typography, breakpoints } = theme;
+  const isUpLg = useMediaQuery(theme.breakpoints.up("lg"));
+
+  const [hoverGeo, setHoverGeo] = useState(null);
+
+  const continentLevelZoom = isUpLg ? 2.4 : 2.1;
+  const countryLevelZoom = isUpLg ? 6 : 5.25;
+  const zoom = level === "continent" ? continentLevelZoom : countryLevelZoom;
+
+  return (
+    <Box
+      sx={{
+        position: "relative",
+      }}
+    >
+      <Box
+        sx={{
+          position: "absolute",
+          zIndex: -1,
+          height: typography.pxToRem(468),
+          width: "100%",
+          [breakpoints.up("md")]: {
+            height: typography.pxToRem(456),
+          },
+          [breakpoints.up("lg")]: {
+            height: typography.pxToRem(600),
```

Comment:
`sx` can handle breakpoints without the explicit use of `theme.breakpoints`
```suggestion
          width: "100%",
          height: { xs: "468px", md: "456px", lg: "600px" },
```